defaults/main.yml

```diff
@@ -45,8 +45,9 @@ datadog_agent_allow_downgrade: no
 # * set 'datadog_agent_allow_downgrade' to yes
 datadog_agent6: no
 
-# beta APT repo where datadog-agent v6 packages are available
+# beta repos where datadog-agent v6 packages are available
 datadog_agent6_apt_repo: "deb http://apt.datadoghq.com beta main"
+datadog_agent6_yum_repo: "https://yum.datadoghq.com/beta/{{ ansible_userspace_architecture }}/"
```

Comment:
We don't plan on shipping 32 bits packages for the agent6. I think we could still keep the `ansible_userspace_architecture`, it will result in an error when ansible is run on unsupported architecture.

Reply:
Do you want me to add a check for that or something?

Reply:
I think we're good with having an error at install time. I just wanted to leave a comment here for future reference.